gtest_parallel_mocks.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python2
```

Comment:
This doesn't need to be executable so you can remove this line, right? (probably applies for all but `gtest-parallel`)